smart-cards/src/main/java/gov/va/api/health/smartcards/Dstu2MetadataController.java

```diff
@@ -0,0 +1,119 @@
+package gov.va.api.health.smartcards;
+
+import static java.util.stream.Collectors.toList;
+
+import gov.va.api.health.dstu2.api.datatypes.ContactPoint;
+import gov.va.api.health.dstu2.api.elements.Reference;
+import gov.va.api.health.dstu2.api.resources.Conformance;
+import java.util.List;
+import java.util.stream.Stream;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(
+    value = {"/dstu2/metadata"},
+    produces = {"application/json", "application/fhir+json"})
+@AllArgsConstructor(onConstructor = @__({@Autowired}))
+public class Dstu2MetadataController {
+
+  private static final String NAME = "API Management Platform | Smart Cards - R4";
+
+  private final BuildProperties buildProperties;
+
+  private final LinkProperties pageLinks;
+
+  private static List<Conformance.Contact> contact() {
+    return List.of(
+        Conformance.Contact.builder()
+            .name("API Support")
+            .telecom(
+                List.of(
+                    ContactPoint.builder()
+                        .system(ContactPoint.ContactPointSystem.email)
+                        .value("api@va.gov")
+                        .build()))
+            .build());
+  }
+
+  private static List<Conformance.RestResource> resources() {
+    return Stream.of(
+            SupportedResource.builder()
+                .type("Parameters")
+                .profileUrl("https://www.hl7.org/fhir/r4/parameters.html")
+                .build())
+        .map(SupportedResource::asResource)
+        .collect(toList());
+  }
+
+  private static List<Conformance.Rest> rest() {
+    return List.of(
+        Conformance.Rest.builder().mode(Conformance.RestMode.server).resource(resources()).build());
+  }
+
+  private Conformance.Implementation implementation() {
+    return Conformance.Implementation.builder().description(NAME).url(pageLinks.r4Url()).build();
+  }
+
+  @GetMapping
+  Conformance read() {
+    return Conformance.builder()
+        .resourceType("Conformance")
```

Comment:
I was going to point out that it is redundant to specify this type... but in fact `Conformance` is a dstu2 resource that does not include defaulting for `resourceType`. Let's open a small PR against `health-apis-fhir-resources` to set default `resourceType` for `Conformance`